src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable %s." ), mon->name().c_str());
+            iexamine::milk_source( mon );
```

Comment:
>It should be safe to assume that all the cows that survived do fear anything that looks like a human.

Actually, researched more on cow taming and things like this pops up a lot https://www.youtube.com/watch?v=vXw620-Tc9Q.

Is there any way to get an item's ID? I can't find any method that does it on item.h, I may have to make one myself.